addons/stock/wizard/make_procurement_product.py

```diff
@@ -36,6 +38,30 @@ def onchange_product_id(self, cr, uid, ids, prod_id):
         'qty': lambda *args: 1.0,
     }
 
+    def fields_view_get(self, cr, uid, view_id=None, view_type='form', context=None, toolbar=False, submenu=False):
```

Comment:
@tde-banana-odoo ,
<code>make_procurement</code> model has only <code>product_id</code> and this wizard use for only two view <code>product_template</code> and <code>product_product</code>. 
As per spec if model of view is <code>product.template</code>, then show only template's variants in wizard,
If template has one variant, then product_id field should invisible on wizard,
If model of view is <code>product.product</code> then product_id field should read-only on wizard,